reference/src/layout/structs-and-tuples.md

```diff
@@ -122,6 +122,7 @@ compiler will not reorder it, to allow for the possibility of
 unsizing. E.g., `struct Foo { x: u16, y: u32 }` and `struct Foo<T> {
 x: u16, y: T }` where `T = u32` are not guaranteed to be identical.
 
+<<<<<<< HEAD
```

Comment:
Something went very wrong here. Please resolve the conflicts.

Reply:
I git pull from my github branch after merging some suggestions into my local branch which had new commits and modifications and everything broke :/

Reply:
Yeah, one has to remember to pull after applying suggestions (and to push before). I made that mistake, too...